dit/dit.py

```diff
@@ -795,13 +792,14 @@ def set(self, argv):
 
     def configure(self, argv):
         rebuild_index = False
+        directory = "~/.dit"
```

Comment:
This property is hidden here on this method. Although not used in any other location, it should be defined as a class attribute to make it more visible.